### DISCUSSION TITLE
git 84821db 后可能的脚本不兼容

### DISCUSSION THREAD

**Discussion by dyphire** [hooke007/mpv_PlayKit#101] [mpv 通用]
自mpv https://github.com/mpv-player/mpv/commit/84821dbcb6d9e16b8f11da2135208e4f3e66fcd0 此提交后的构建版本将破坏缩略图脚本
![](https://cdn.jsdelivr.net/gh/dyphire/PicGo/img/2022/03/24/20220324200714.png)

**Comment by hooke007** [comment]
不是很理解上游是在做什么，保险起见我要排查一下所有脚本的常用功能是否正常

thumbnailer_worker.lua 已经修了，暂无发现其它不兼容的案例

**Comment by natural-harmonia-gropius** [comment]
为了符合[参考手册](https://www.lua.org/manual/5.1/manual.html#:~:text=If%20the%20function%20was%20defined%20in%20a%20string%2C%20then%20source%20is%20that%20string.%20If%20the%20function%20was%20defined%20in%20a%20file%2C%20then%20source%20starts%20with%20a%20%27%40%27%20followed%20by%20the%20file%20name.)改了路径的处理方式
